src/main/java/net/glowstone/scheduler/GlowScheduler.java

```diff
@@ -74,17 +76,33 @@
      * The primary worlds thread in which pulse() is called.
      */
     private Thread primaryThread;
+    /**
+     * The session registry used to pulse all players.
+     */
+    private final SessionRegistry sessionRegistry;
 
     /**
      * Creates a new task scheduler.
      *
      * @param server The server that will use this scheduler.
-     * @param worlds The {@link WorldScheduler} this scheduler will use for ticking the server's
+     * @param worlds The {@link WorldScheduler} this scheduler will use for ticking the servers
```

Comment:
The previous doc was correct: server's worlds

Reply:
Fixed on both constructors.